pkg/shell/index.html

```diff
@@ -162,6 +164,148 @@ <h4 class="modal-title" id="error-popup-title"></h4>
       </div>
     </div>
 
+    <div class="modal" id="credentials-dialog" tabindex="-1" role="dialog" data-backdrop="static">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title" translatable="yes">Authentication</h4>
+                </div>
+                <div class="modal-body modal-scrollable">
+                    <table class="listing credential-listing">
+                        <thead>
+                            <tr id="credential-authorize">
+                                <td translatable="yes" colspan="2">Use my password for privileged tasks and to connect to other machines</td>
+                                <td class="listing-actions">
+                                    <button class="btn btn-default pficon pficon-close" id="deauthorize-item">
+                                    </button>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td translatable="yes" colspan="2">Use the following keys to authenticate against other systems</td>
+                                <td class="listing-actions">
+                                    <button class="btn btn-primary fa fa-plus disabled">
```

Comment:
I'm guessing generating a new key is a different pull request. Should we remove this button until then?